recipes/icub-main/build_py.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+if [[ "${target_platform}" == osx-* ]]; then
+    # See https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
+    CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
```

Comment:
Does this have an effect? My basic bash knowledge tells me that this only exposes the updates CXXFLAGS to the script, but not to processes spawned by it.

Reply:
Good point. To be honest, this piece of code is a bit of cargo cult, it may not be necessary, let me try to remove it.

Reply:
Yes, it turns out that anyhow these was not needed, I removed and the CI is happy. I will rebase the commit, but the PR is anyhow ready for review.